src/frame.rs

```diff
@@ -676,6 +704,19 @@ impl Frame {
             Frame::HandshakeDone => {
                 1 // frame type
             },
+
+            Frame::AckFrequency {
+                sequence_number,
+                packet_tolerance,
+                update_max_ack_delay,
+                ignore_order,
+            } => {
+                1 + // frame type
+                octets::varint_len(*sequence_number) + // sequence_number
+                octets::varint_len(*packet_tolerance) + // packet_tolerance
+                octets::varint_len(*update_max_ack_delay) + // update_max_ack_delay
+                octets::varint_len(*ignore_order as u64)
```

Comment:
since ignrore_order is just a bool, you can probably just add 1 here

Reply:
Fixed.